slasher/rpc/server.go

```diff
@@ -55,12 +56,10 @@ func (ss *Server) IsSlashableBlock(ctx context.Context, psr *ethpb.ProposerSlash
 
 // SlashableProposals is a subscription to receive all slashable proposer slashing events found by the watchtower.
 func (ss *Server) SlashableProposals(req *types.Empty, server ethpb.Slasher_SlashableProposalsServer) error {
-	//TODO(3133): implement stream provider for newly discovered listening to slashable proposals.
```

Comment:
why remove this ? it is still unimplemented

Reply:
i guess bad merge

Reply:
getting it back